### PR TITLE
Update worldbank instance deployment config

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: deploy/helm_charts/envs/worldbank.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ mixer:
   useBaseBigtable: false
   useBranchBigtable: false
   useCustomBigtable: true
-  remoteMixerDomain: https://api.datacommons.org
+  remoteMixerDomain: https://autopush.api.datacommons.org
```
Comment (keyurva) [inline deploy/helm_charts/envs/worldbank.yaml:35]: nit: Consider adding a note to update this URL before launch.

**Review by keyurva — APPROVED** [review]
Thanks.

File: deploy/helm_charts/envs/worldbank.yaml (hunk 1 above)
Comment (shifucun) [reply deploy/helm_charts/envs/worldbank.yaml:35]: Done

**Comment by shifucun** [comment]
thanks!

**Pull request merged**